bigchaindb/backend/query.py

```diff
@@ -318,3 +318,16 @@ def get_unvoted_blocks(connection, node_pubkey):
     """
 
     raise NotImplementedError
+
```

Comment:
the `get_txids_by_asset_id` singledispatch is no longer being implemented so we can just remove it